ds_judgements_public_ui/templates/judgment/detail.html

```diff
@@ -46,13 +46,20 @@
     {{ content }}
   {% endif %}
   {% include "includes/judgment_end_document_marker.html" %}
-  <div id="js-document-navigation-links-end"
-       class="document-navigation-links {% if context.query %}with-query{% endif %}">
-    <span class="up-wrapper">
-      <a id="start-of-document-link"
-         class="up document-navigation-link"
-         href="#start-of-document">Back to start</a>
-    </span>
+  <div class="document-navigation" id="js-document-navigation">
+    <div class="document-navigation__summary">
+      <h2>{{ context.page_title }}</h2>
+      <p>{{ context.judgment_ncn }}</p>
+    </div>
+
+    <div id="js-document-navigation-links-end"
+         class="document-navigation-links {% if context.query %}with-query{% endif %}">
```

Comment:
this will need to be just "query" -- Nick's removing all the 'context-named-context' bits

Reply:
Have rebased against main and sorted out the context stuff :) thanks for pointing out